perception/bytetrack/include/bytetrack/bytetrack_node.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BYTETRACK__BYTETRACK_NODE_HPP_
+#define BYTETRACK__BYTETRACK_NODE_HPP_
+
+#include <bytetrack/bytetrack.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <tier4_perception_msgs/msg/detected_objects_with_feature.hpp>
+#include <tier4_perception_msgs/msg/dynamic_object_array.hpp>
+
+#include <cv_bridge/cv_bridge.h>
+
+#include <chrono>
+#include <fstream>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace bytetrack
+{
+using LabelMap = std::map<int, std::string>;
+
+class ByteTrackNode : public rclcpp::Node
+{
+public:
+  explicit ByteTrackNode(const rclcpp::NodeOptions & node_options);
+
+private:
+  void OnConnect();
+  void OnRect(const tier4_perception_msgs::msg::DetectedObjectsWithFeature::ConstSharedPtr msg);
+
```

Comment:
In the coding guideline, it recommend  to use lower snake case for function.
Also, the bytetrack lib use lower snake case. So, I think it is better to change the names, how about?

Reply:
I really appreciate your letting me know the guideline! Because there is no strong intention to use the camel case style, I've rewritten the function names in the modification of 84ae46ef142599708c4a61d3ba9f1b7a8368c79b so that they follow the coding guideline.